packages/techdocs-common/api-report.md

```diff
@@ -93,6 +94,9 @@ export const getLastCommitTimestamp: (repositoryLocation: string, logger: Logger
 // @public (undocumented)
 export const getLocationForEntity: (entity: Entity) => ParsedLocationAnnotation;
 
+// @public
+export const getSourceLocation: (entity: Entity) => string | undefined;
```

Comment:
Not sure why all the helpers are exported. I couldn't find any use in the backstage repo.

Reply:
😬